helpdesk_mgmt/__manifest__.py

```diff
@@ -14,7 +14,7 @@
     "SDi Soluciones, "
     "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/helpdesk",
-    "depends": ["mail", "portal", "website"],
+    "depends": ["mail", "portal", "website", "stock"],
```

Comment:
This should be `sale_stock` as the relation between sale and stock is nonexistent without it, ie: sale order objects do not have the `picking_ids` reference without it.